cr_electronic_invoice/data/uom_data.xml

```diff
@@ -1,29 +1,305 @@
+            <field name="category_id" ref="uom.product_uom_categ_unit"/>
+            <field name="name">Unidad</field>
+            <field name="factor" eval="1"/>
+            <field name="rounding" eval="1"/>
+            <field name="code">Unid</field>
+            <field name="uom_type">reference</field>
+            <field name="active">True</field>
+        </record>
+
+        <record id="product_uom_mcubico" model="uom.uom">
+            <field name="category_id" ref="uom.product_uom_categ_vol"/>
             <field name="name">Metro Cubico</field>
             <field name="factor" eval="1.0"/>
             <field name="rounding" eval="0.01"/>
             <field name="code">m3</field>
-            <field name="uom_type">reference</field>
+            <field name="uom_type">bigger</field>
         </record>
         <record id="product_uom_sp" model="uom.uom">
-            <field name="category_id" ref="uom.product_uom_categ_sp"/>
+            <field name="category_id" ref="product_uom_categ_service"/>
             <field name="name">Servicios Profesionales</field>
             <field name="factor" eval="1.0"/>
             <field name="rounding" eval="0.01"/>
             <field name="code">Sp</field>
             <field name="uom_type">reference</field>
         </record>
+
+        <record id="product_uom_spe" model="uom.uom">
+            <field name="category_id" ref="product_uom_categ_service"/>
+            <field name="name">Servicios Personales</field>
+            <field name="factor" eval="1.0"/>
+            <field name="rounding" eval="0.01"/>
+            <field name="code">Spe</field>
+            <field name="uom_type">smaller</field>
+        </record>
+
+         <record id="product_uom_st" model="uom.uom">
+            <field name="category_id" ref="product_uom_categ_service"/>
+            <field name="name">Servicios Técnicos</field>
+            <field name="factor" eval="1.0"/>
+            <field name="rounding" eval="0.01"/>
+            <field name="code">St</field>
+            <field name="uom_type">smaller</field>
+        </record>
+
+         <record id="product_uom_os" model="uom.uom">
+            <field name="category_id" ref="product_uom_categ_service"/>
+            <field name="name">Otro Tipo de Servicio</field>
+            <field name="factor" eval="1.0"/>
+            <field name="rounding" eval="0.01"/>
+            <field name="code">Os</field>
+            <field name="uom_type">smaller</field>
+        </record>
+
+        <record id="product_uom_i" model="uom.uom">
+            <field name="category_id" ref="uom.product_uom_categ_unit"/>
+            <field name="name">Intereses</field>
+            <field name="factor" eval="1.0"/>
+            <field name="rounding" eval="0.01"/>
+            <field name="code">I</field>
+            <field name="uom_type">bigger</field>
+        </record>
+
+        <record id="product_uom_com" model="uom.uom">
```

Comment:
@WhiteTiger08 
* "UoM category Rent should have a reference unit of measure. If you just created a new category, please record the 'reference' unit first." --> [Travis CI](https://travis-ci.com/odoocr/l10n_cr/jobs/209474052#L1856)

Reply:
Si lo tiene, pero no estaba en orden, ya lo invertí

Reply:
"You cannot change the ratio of this unit of mesure as some products with this UoM have already been moved or are currently reserved." --> [Travis CI](https://travis-ci.com/odoocr/l10n_cr/jobs/209528001#L1543)